collections/_pages/features.html

```diff
@@ -86,7 +86,7 @@
         highlight: false
         featureli: true
     call_to_action:
-      link: /#getting-started
+      link: https://docs.meshery.io/functionality/filter-management#q-what-is-wasm-
```

Comment:
```suggestion
      link: https://docs.meshery.io/functionality/filter-management
```
Should I make the link as above?